changelog/unreleased/issue-340

```diff
@@ -0,0 +1,10 @@
+Bugfix: Add support for Volume Shadow Copy Service (VSS) on windows
```

Comment:
This should probably start with "Enhancement" instead of "Bugfix"?

Reply:
Fixed in https://github.com/restic/restic/pull/2274/commits/ea769e845ff18c558c098f265b798918e2a4a28d